doc/administrator/galaxy/pipelines/mentalist/index.md

```diff
@@ -11,7 +11,7 @@ This workflow uses the software [MentaLiST][] for typing of microbial samples di
 
 | Tool Name                      | Owner    | Tool Revision | Toolshed Installable Revision | Toolshed             |
 |:------------------------------:|:--------:|:-------------:|:-----------------------------:|:--------------------:|
-| **mentalist**                  | dfornika | 1d9e3950ce61  | 0 (2017-12-20)                | [Galaxy Main Shed][] |
+| **mentalist**                  | dfornika | 4ea71c8e2db0  | 3 (2018-04-11)                | [Galaxy Main Shed][] |
```

Comment:
Could you also possibly add to this file as a note that if you end up with duplicate database tools, you can always remove the older ones by deactivating the tools.

That is, after installing mentalist `0.1.5`, the data manager screen will look like:

![image](https://user-images.githubusercontent.com/200517/38955677-fbad9f24-431a-11e8-9afb-246cdeadebf5.png)

You can remove the duplicates here by deactivating mentalist `0.1.3`.

![image](https://user-images.githubusercontent.com/200517/38955773-36ca4cb0-431b-11e8-8fdc-8ec7779e562a.png)